scripts/filesize.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 check_command() {
-	which "$1" >/dev/null 2>&1
+	command -v "$1" >/dev/null 2>&1
```

Comment:
AFAICT, `command -v` doesn't print any message on errors, so the `2>&1` can be dropped (in multiple scripts)